Pipfile

```
@@ -4,7 +4,7 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-aiohttp = "*"
+aiohttp = ">=3.4,<=4"
```

```suggestion
aiohttp = ">=3.4,<4"
```

Would it allow version `4.0.0` as it is?